index.php

```diff
@@ -0,0 +1,2 @@
+<?php
+header('Location: index.html');
```

Comment:
Really... a php file? also no closing ?> tag (or am I out of date on making sure you always close tags is this some new standard?